packages/backend/src/api/controllers/status/Status.ts

```diff
@@ -0,0 +1,6 @@
+export interface Status {
+  name: string
+  timestamp: number | undefined
+  value: string | undefined
```

Comment:
Why `undefined` instead of optional values?

Reply:
`timestamp: latest?.timestamp.toNumber()` during object creation we are using .? operator so it will set undefined. Having optional parameter is more clean?

Reply:
"clean" is not very specific, but for sure it is less verbose. It has a different meaning though, because optional parameter does not have to be explicitly set as `undefined` whereas with `| undefined`  it does have to. This property is neither good nor bad - it just depends on what we want to achieve. As far as my understanding goes, here we don't gain anything from explicit union with `undefined`, but the code becomes more verbose.